docs/bcfusion/overview.md

```diff
@@ -0,0 +1,39 @@
+# Overview
+
+BNB Beacon Chain is a blockchain developed by the BNB Chain community that implements a vision of a decentralized
+exchange (DEX) for digital assets. Besides this, Beacon Chain and BSC is a dual-chain structure: Beacon Chain helps to
+enhance the security of BSC as a staking and governance layer. With the rise of various other forms of Dex, order-book
+based decentralized exchange was decommissioned
+in [BEP151](https://github.com/bnb-chain/BEPs/blob/master/BEPs/BEP151.md). With the quick evolution of BSC, the Beacon
+Chain has become a burden. The cross-chain bridge that connects the two chains slows down the development iteration and
+always exposes BNB to a certain level of security vulnerabilities. It's time to take a step further and migrate the
+functionality of Beacon Chain to BSC, **allowing Beacon Chain to retire**.
+
+![img](../assets/bcfusion/phases.png)
+
+There will be several pahses to retrie Beacon Chain:
+
+- **First Sunset Fork** - Some types of Beacon chain transactions will be disabled, for example, TimeLockMsg,
+  TimeRelockMsg,
+  FreezeMsg, IssueMsg, MintMsg, IssueMiniMsg, HTLTMsg, DepositHTLTMsg, MsgCreateValidatorOpen,
+  MsgCreateSideChainValidator, MsgCreateSideChainValidatorWithVoteAddr, MsgEditSideChainValidatorWithVoteAddr,
+  MsgSideChainDelegate, MsgSideChainReDelegate. (Estimated time on mainnet: 2024 Feb)
+- **BSC Feynman Hardfork** - Native validators and staking, native goverenance will be enabled on BNB Smart Chain. (
```

Comment:
+ `The BSC validator/delegator can start migration after the Feynman upgrade.`

Reply:
updated